tensorflow/python/training/saver.py

```diff
@@ -1597,9 +1597,9 @@ def save(self,
         [Stripping Default-Valued Attributes](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md#stripping-default-valued-attributes).
 
     Returns:
-      A string: path prefix used for the checkpoint files. If checkpoint
-        format is V1 and the saver is sharded, this string ends with:
-         '-?????-of-nnnnn' where 'nnnnn' is the number of shards created.
+      A string: path prefix used for the checkpoint files.  If the saver is
+        sharded, this string ends with: '-?????-of-nnnnn' where 'nnnnn'
+        is the number of shards created.
```

Comment:
Revert this file

Reply:
This is actually needed. We need to roll this back and propagate this rollback also internally to fix the last pull from github.
This file and evaluation_test.py changes are a part of the rollback, I manually verified those two.

Reply:
Okay sounds good.